test/GameState.test.js

```diff
@@ -0,0 +1,58 @@
+const expect = require("chai").expect;
+const { GameState } = require("../entities/GameState");
+const { createPlayer } = require("./testutils");
+
+describe("GameState", () => {
+  let gameState;
+  beforeEach(() => {
+    gameState = new GameState();
+  });
+  it("should load default values", () => {
```

Comment:
I don't think that we need this here; if something consumes the GameState and requires a particular configuration upon its consumption I think this would make sense as an integration test, but not here as a unit

Reply:
as in, this specific test?